src/jsp/jsp_cl.cpp

```diff
@@ -1102,6 +1157,72 @@ drop_stored_procedure (const char *name, SP_TYPE_ENUM expected_type)
   return err;
 }
 
+/*
+ * drop_stored_procedure_code -
+ *   return: Error code
+ *   name(in): jsp name
+ *
+ * Note:
+ */
+
+static int
+drop_stored_procedure_code (const char *name)
+{
+  MOP code_mop, owner;
+  DB_VALUE owner_val, generated_val;
+  int save;
+  int err;
+
+  AU_DISABLE (save);
+
+  db_make_null (&owner_val);
+
+  code_mop = jsp_find_stored_procedure_code (name);
+  if (code_mop == NULL)
+    {
+      assert (er_errid () != NO_ERROR);
+      err = er_errid ();
+      goto error;
+    }
+
+  err = db_get (code_mop, SP_ATTR_OWNER, &owner_val);
+  if (err != NO_ERROR)
+    {
+      goto error;
+    }
+  owner = db_get_object (&owner_val);
+
+  if (!ws_is_same_object (owner, Au_user) && !au_is_dba_group_member (Au_user))
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_SP_DROP_NOT_ALLOWED_PRIVILEGES, 0);
+      err = er_errid ();
+      goto error;
+    }
+
+  err = db_get (code_mop, SP_ATTR_IS_SYSTEM_GENERATED, &generated_val);
+  if (err != NO_ERROR)
+    {
+      goto error;
+    }
+
+  if (!DB_IS_NULL (&generated_val) && 1 == db_get_int (&generated_val))
```

Comment:
trivial;
 I think it would be better to use the defined value rather than 1.

Reply:
In system catalog, Boolean column type does not exists, so it is represented as Integer type with 1 and 0 values. I think we need a definition such as `BOOLEAN_TRUE` and `BOLLEAN_FALSE` in system_catalog module. IMO, it is not scope of this PR, I'd like to add the definition in another system catalog refactoring issue.
Thank you for helpful refactoring tip.